docs/contributing_guides/PULL_REQUEST.md

```diff
@@ -0,0 +1,67 @@
+# Guide to pull requests (PR)
+
+## Why is it important to write good PRs?
+
+Good PRs are important because they help maintainers understand the changes you've made to the codebase. They also help you keep track of what you've done.
+
+<!-- TOC -->
+
+- [🐒 Workflow on how to create a PR](#workflow-on-how-to-create-a-pr)
+- [✏️ How to write a PR](#-how-to-write-a-pr)
+  - [Title](#title)
+  - [Description](#description)
+  - [One commit per PR](#one-commit-per-pr)
+  - [Title and description of the commit](#title-and-description-of-the-commit)
+- [Resources](#resources)
+
+<!-- /TOC -->
+
+## 🐒 Workflow on how to create a PR
+<a id="markdown-workflow-on-how-to-create-a-pr" name="workflow-on-how-to-create-a-pr"></a>
+
+If you're interested in contributing code, updating doc etc., please follow these steps:
+
+1. Fork the repository on GitHub.
+2. Clone your forked repository to your local machine.
+3. Create a new branch for your changes. Please read our [guide to naming a 🌳 branch](https://github.com/r3bl-org/r3bl-open-core/blob/fe8ef6b8bbc2b1f73730659b1863216c8b8f0909/docs/contributing_guides/BRANCH.md).
+4. Make your changes and commit them with a descriptive commit message. Please read our [guide to writing a 📝 commit message](https://github.com/r3bl-org/r3bl-open-core/blob/fe8ef6b8bbc2b1f73730659b1863216c8b8f0909/docs/contributing_guides/COMMIT_MESSAGE.md).
+5. Write tests when relevant.
+6. Run tests: `nu run.nu test`.  Take a look at [other scripts you can run](https://github.com/r3bl-org/r3bl-open-core/tree/main/tui#run-the-demo-locally).
```

Comment:
While it is accurate to run `nu run.nu <command>` it is probably easier to run `./run.nu <command` on macOS & Linux. The "long form" of the command runs on windows, macos, and linux, so it it might be more cross platform. I'm not sure if we should just write the commands in the short form since we mostly use Linux?

Reply:
That makes sense.